ursula_cli/shell.py

```diff
@@ -527,6 +542,14 @@ def parse_args():
         '--heat-stack-name', default=None,
         help='Name of the heat stack when heat provisioner is used',
     )
+    parser.add_argument(
+        '--heat-stack-update', default=False, action='store_true',
+        help='Update the heat stack',
+    )
+    parser.add_argument(
+        '--heat-parameters', default=None,
+        help='Semicolon separated list of KEY=VALUE parameters for heat',
```

Comment:
From looking at this again, looks like `format_parameters` will take a list and try to split each element of the list on the `=`.   If that's the case, and the arguments are lists of `key=value` pairs, then maybe use `action='append'` and specify multiple times for each heat param?  or maybe we `.split()` on spaces before passing into `format_parameters`